integreat_compass/cms/templates/index.html

```diff
@@ -3,16 +3,128 @@
+                        <ul class="items-center w-full border border-gray-500 rounded-full flex shadow-inner">
+                            {% for mode_type in offer_mode_types %}
+                                <li class="w-full text-center">
+                                    <div class="items-center">
+                                        <input id="mode-type-{{ mode_type.0 }}"
+                                               class="filter-element hidden peer"
+                                               type="checkbox"
+                                               name="mode-type"
+                                               {% if mode_type.0 in filtered_mode_types %}checked{% endif %}
+                                               value="{{ mode_type.0 }}">
+                                        <label for="mode-type-{{ mode_type.0 }}"
+                                               class="w-full duration-100 m-0 p-2 text-lg rounded-full peer-checked:bg-primary peer-checked:text-white">
+                                            {{ mode_type.1 }}
+                                        </label>
+                                    </div>
+                                </li>
+                            {% endfor %}
+                        </ul>
+                    </div>
+                </form>
+            </div>
+        </div>
+        <div class="mr-12 w-full lg:ml-[25vw] lg:w-9/12 justify-end lg:pr-0">
+            <div class="pl-12">
+                <div class="mb-4">
+                    <div class="flex items-center">
+                        <h2 class="text-2xl font-semibold">{% translate "Offers" %}</h2>
+                        <span class="flex ml-4 text-gray-400 items-center">{{ offers|length }}
+                            {% translate "offers" %}
+                        </span>
+                    </div>
+                </div>
+                <div class="grid lg:grid-cols-2 gap-8">
+                    {% if offers %}
+                        {% for offer in offers %}
+                            <div id="offer-{{ offer.id }}"
+                                 class="single-offer bg-white flex shadow-lg hover:shadow-xl rounded-2xl hover:cursor-pointer">
+                                <div class="w-4/12 flex items-center">
+                                    {% if offer.versions.first.title_image %}
+                                        <img class="rounded-2xl"
+                                             src="{{ offer.versions.first.title_image.url }}"
+                                             width="100%"
+                                             height="100%"
+                                             alt="">
+                                    {% endif %}
+                                </div>
+                                <div class="w-8/12 py-2 px-4">
+                                    <h3 class="font-bold">{{ offer.versions.first.title }}</h3>
```

Comment:
```suggestion
                                    <h3 class="font-bold">{{ offer.public_version.title }}</h3>
```
Sorry I did not notice this before. This has to be done everywhere you use any property of the offer version, since users should never see non-public version content.